src/tests/tribol_mfem_common_plane.cpp

```diff
@@ -199,21 +199,24 @@ class MfemCommonPlaneTest : public testing::TestWithParam<int> {
       tribol::BINNING_GRID
     );
     tribol::registerMfemVelocity(0, v);
-    tribol::setPenaltyOptions(
```

Comment:
Why is this removed?

Reply:
It gets set in `setMfemKinematicConstantPenalty()` and `setMfemKinematicElementPenalty()`